pkg/acsengine/testdata/location/kubernetes.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
check file acsengine.go, to only produce kubeconfig for the given region. Otherwise, fall back to previous behavior.